server/test/server.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require('supertest');
+const app = require('../app');
+
+
+describe(" Test api/items/:id endpoint", function(){
+
+  test("It should respond with 200 with correct id", async (done) => {
+    const response = await request(app).get('/api/items/1');
+    expect(response.statusCode).toBe(200);
+    done();
+
+  });
+
+  test("It should respond  with correct id", async (done) => {
+    const response = await request(app).get('/api/items/2');
+    expect(response.body.itemId).toBe(2);
+    done();
+
+  });
+
+  test("It should  fail to respond  with correct main image", async (done) => {
+     const mainImage = "https://i.picsum.photos/id/99/200/200.jpg";
+    const response = await request(app).get('/api/items/2');
+    expect(response.body.mainImage).not.toBe(mainImage);
+    done();
+
+  });
+
+  test("It should  respond  with correct main image", async (done) => {
+    const mainImage = "https://i.picsum.photos/id/99/200/200.jpg";
+   const response = await request(app).get('/api/items/99');
+   expect(response.body.mainImage).toBe(mainImage);
+   done();
+
+ });
+
+ test("It should  contain correct array items", async (done) => {
+  const mainImage = [
+    "https://i.picsum.photos/id/492/200/200.jpg",
+    "https://i.picsum.photos/id/493/200/200.jpg",
+    "https://i.picsum.photos/id/494/200/200.jpg",
+    "https://i.picsum.photos/id/495/200/200.jpg"

```

Comment:
These images will eventually be uploaded to and served from S3, correct?

Reply:
yeah that is the plan. But right now those are just some random photos.